templates/hpe/index.html

```diff
@@ -381,7 +382,7 @@ <h5 class="p-muted-heading">Contact</h5>
       <p>Contact us for more information about the Canonical and HPE partnership or how to purchase our joint solutions.</p>
       <div class="p-strip">
         <h3 class="p-heading--2">
-          <a href="/contact-us">Send us a message&nbsp;&rsaquo;</a>
+          <a href="/contact-us#get-in-touch">Send us a message&nbsp;&rsaquo;</a>
```

Comment:
This link doesn't have the `js-invoke-modal` class so it doesn't pop up the form on the page but instead redirects to the form on `/contact-us`.